### PR TITLE
Styled the student section

### PR DESCRIPTION
**Pull request by opwildgoose**
Added the basic components of the page. There are parts that still need to be adjusted.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -6,38 +6,40 @@
 	<meta charset="utf-8">
 	<meta name="description" content="Contact page for JumboCode website">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel='stylesheet' href='style.css'/>
 
 	<!-- Favicon -->
 	<!-- <link rel="icon" href="/assets/img/favicon.ico" type="image/x-icon" /> -->
 
-	<!-- Stylesheets -->
```
Comment (olivaceously) [inline contact.html]: Please keep this, and put the `<link>` to the stylesheet below this comment - it's a good way to standardize our code

File: contact.html [hunk 2]
```diff
@@ -53,6 +55,27 @@ <h3 class="head-font"> Facebook </h3>
...
+    
+    <div class="sections">
+        <div class="overview">
+            <h1 class="green">Students,</h1>
+            <h1>looking to join</h1>
+            <h1>JumboCode?</h1>
+        </div>
+        
+        <div class="description">
+            <p>Unfortunately, the current JumboCode seans has already begun, and at this point we are no longer looking to accept any more team members.</p>
```
Comment (olivaceously) [inline contact.html]: typo -> change "seans" to "season"

File: contact.html [hunk 3]
```diff
@@ -53,6 +55,27 @@ <h3 class="head-font"> Facebook </h3>
...
+        
+            <p>But we are always excited about the next round of projects, and would love to have your support soon, regardless of your skill leve. Please like us on Facebook and keep an eye out for our next GIM - that's when we'll be announcing the upcoming projects and recruting new team members.</p>
```
Comment (olivaceously) [inline contact.html]: Also, "recruting" -> "recruiting"

File: style.css [hunk 4]
```diff
@@ -9,7 +9,90 @@ p {
...
+
+.facebook {
+    padding: 20px;
+    font-size: 30px;
+    width: 50px;
+    text-align: center;
+    text-decoration: none;
+    background: #3B5998;
+    color: white;
+}
```
Comment (olivaceously) [inline style.css]: please add a new line to the bottom of your file

**Comment by olivaceously** [comment]
Can you remove the binary image file? I will ask Matt for the image that we are supposed to be using in the background and upload it to your branch. Look for my slack message!

**Comment by opwildgoose** [comment]
Added a new branch no_join_us with requested changes

**Comment by olivaceously** [comment]
This PR is no longer needed - contact page is being styled by Monique/Noah.

**Pull request closed without merging**